src/libs/PaymentUtils.js

```diff
@@ -29,28 +29,54 @@ function hasExpensifyPaymentMethod(cardList = [], bankAccountList = []) {
  * Get the PaymentMethods list
  * @param {Array} bankAccountList
  * @param {Array} cardList
+ * @param {Object} personalBankAccount
  * @param {String} [payPalMeUsername='']
  * @param {Object} userWallet
  * @returns {Array<PaymentMethod>}
  */
-function formatPaymentMethods(bankAccountList, cardList, payPalMeUsername = '', userWallet) {
+function formatPaymentMethods(bankAccountList, cardList, personalBankAccount = {}, payPalMeUsername = '', userWallet) {
```

Comment:
We're removing the data massaging happening in this function in this PR: https://github.com/Expensify/App/pull/10447

Can we also ensure we don't do any massaging of the data for the PBA?

Reply:
I love that we are not going to do any data massaging, however, this is all data that lives in onyx. We are not getting any new data from the server. We are just taking the plaid and user input and holding on to it while they figure out the errors.